environment/src/main/java/com/sequenceiq/environment/parameters/dao/domain/AzureParameters.java

```diff
@@ -36,6 +36,9 @@
     @SecretValue
     private Secret encryptionKeyUrl = Secret.EMPTY;
```

Comment:
For lines 34--37: As discussed offline, please also add the new (renamed) column `encryption_key_url`. The class field for that can be left with thee old name here (which is spelled right), and the legacy `@Column(name = "encryption_keyUrl")` column def shall be applied to a new field `private Secret encryptionKeyUrlDummy` or something like that. `encryptionKeyUrlDummy` shall also receive the annotation `@Deprecated` with the message saying what to use instead..